src/hp/derp/derp_cfg.rs

```diff
@@ -0,0 +1,62 @@
+// from tailscale/tailcfg/derpmap.go
```

Comment:
why call this file cfg when the source is map?

Reply:
good point